Library/Homebrew/dev-cmd/ruby.rb

```diff
@@ -1,13 +1,28 @@
 #:  * `ruby` [<ruby options>]:
 #:    Run a Ruby instance with Homebrew's libraries loaded.
-#:    For example:
-#       brew ruby -e "puts :gcc.f.deps"
-#       brew ruby script.rb
+#:
+#:    *Example:* `brew ruby -e "puts :gcc.f.deps"` or `brew ruby script.rb`
+
+require "cli_parser"
 
 module Homebrew
   module_function
 
+  def ruby_args
```

Comment:
Same reason as mentioned `prof.rb`. Cannot use `CLI::Parser` yet.

Reply:
`brew ruby` doesn't have to take arbitrary arguments; we can document the `ruby` arguments we want to accept (like `ls`)?

Reply:
Yeah we can accept a fix set of options. What options are we planning on supporting for `brew ruby`?

Reply:
@GauthamGoli I think `-e` and `-v` are fine for now.

Reply:
@EricFromCanada Please declare these options.

Reply:
Have a look.